.github/fabricbot.json

```diff
@@ -1,1182 +0,0 @@
-          ],
-          "mentionees": [
-            "dotnet/area-system-io"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Diagnostics.Process"
-          ],
-          "mentionees": [
-            "dotnet/area-system-diagnostics-process"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Console"
-          ],
-          "mentionees": [
-            "dotnet/area-system-console"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.IO.Compression"
-          ],
-          "mentionees": [
-            "dotnet/area-system-io-compression"
-          ]
-        }
-      ],
-      "replyTemplate": "Tagging subscribers to this area: ${mentionees}\nSee info in [area-owners.md](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you want to be subscribed.",
-      "enableForPullRequests": true
-    },
-    "disabled": false
-  },
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Add untriaged label to new/reopened issues without a milestone",
-      "actions": [
-        {
-          "name": "addLabel",
-          "parameters": {
-            "label": "untriaged"
-          }
-        }
-      ],
-      "eventType": "issue",
-      "eventNames": [
-        "issues"
-      ],
-      "conditions": {
-        "operator": "and",
-        "operands": [
-          {
-            "operator": "or",
-            "operands": [
-              {
-                "name": "isAction",
-                "parameters": {
-                  "action": "opened"
-                }
-              },
-              {
-                "name": "isAction",
-                "parameters": {
-                  "action": "reopened"
-                }
-              },
-              {
-                "name": "removedFromMilestone",
-                "parameters": {}
-              }
-            ]
-          },
-          {
-            "name": "isOpen",
-            "parameters": {}
-          },
-          {
-            "operator": "not",
-            "operands": [
-              {
-                "name": "isInMilestone",
-                "parameters": {}
-              }
-            ]
-          },
-          {
-            "operator": "not",
-            "operands": [
-              {
-                "name": "hasLabel",
-                "parameters": {
-                  "label": "untriaged"
-                }
-              }
-            ]
-          }
-        ]
-      }
-    }
-  },
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Remove untriaged label from issues when closed or added to a milestone",
```

Comment:
This task didn't get ported over, presumably because it used the `addedToMilestone` action which is not supported.